### DISCUSSION TITLE
How to add 3 digits after the decimal point in the amount

### DISCUSSION THREAD

**Discussion by Khathijasafana** [baimard/gestion#418] [Features request]
Hi all,

Can anyone suggest ,how can we add 3 digits after decimal point  in the Total price and VAT

**Comment by baimard** [comment]
Hello, it's not allowed, only 2 digits. Maybe it's an idea to make it as a parameter.

You need this functionnality ?

**Comment by Khathijasafana** [comment]
Yes correct @baimard :-)
